src/variants/beamos/config

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 # variant mrbeam, flavor default: override hostname & custom config setting
 export DIST_NAME=BeamOS
+export DIST_VERSION=0.18.2
```

Comment:
where exactly this variable will be written to a file ?

Reply:
this will be done in the [CustomPiOs](https://github.com/mrbeam/CustomPiOS) -> [base/start_chroot_script](https://github.com/mrbeam/CustomPiOS/blob/mrbeam/src/modules/base/start_chroot_script)
```
# Store dist version
echo "$DIST_VERSION" > /etc/${DIST_NAME,,}_version
```